README.md

```diff
@@ -1,35 +1,58 @@
 # MkDocs
 
-Project documentation with Markdown.
```

Comment:
Curious why you deleted our tagline?

To be clear, I have no attachment to the tagline, but have left it in place as it is the tagline for the project.

Reply:
That should be a tagline? Like I seriously didn't see it as one...